grisera_api/activity_execution/activity_execution_service_relational.py

```diff
@@ -132,5 +137,5 @@ def get_multiple_with_foreign_id(self, foreign_id: Union[int, str], depth: int =
                 for activity_execution in activity_executions:
                     activity_execution["arrangements"] = arrangement_service_relational.get_arrangement(
                         activity_execution["arrangement_id"], depth - 1, self.table_name)
-        return ActivityExecutionsOut(activity_executions=activity_executions)
-    
+        return [ActivityExecutionOut(**execution) for execution in activity_executions]
```

Comment:
Nit: could just return activity_executions

Reply:
this cannot produce an error